WORKSPACE.bazel

```diff
@@ -0,0 +1,254 @@
+workspace( name = "coda" )
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_repository")
+
+################################################################
+#### OCAML ####
+git_repository(
+    name = "obazl_rules_ocaml",
+    remote = "https://github.com/obazl/rules_ocaml",
+    branch = "dev",
+    # commit = "feef897197b36b14b65ffdf00b9badcbdb8f42f4",
+    # shallow_since = "1593623637 -0500"
+)
+
+load("@obazl_rules_ocaml//ocaml:bootstrap.bzl",
+     "ocaml_configure",
+     "opam_configure",
+     "ocaml_register_toolchains")
+
+opam_configure()
+
+ocaml_configure()
+
+ocaml_register_toolchains(installation="host")
+
+################################################################
+git_repository(
+    name = "rules_foreign_cc",
+    remote = "https://github.com/bazelbuild/rules_foreign_cc",
+    commit = "74b146dc87d37baa1919da1e8f7b8aafbd32acd9",
+    shallow_since = "1588931020 +0200"
+    # strip_prefix = "rules_foreign_cc-master",
+    # url = "https://github.com/bazelbuild/rules_foreign_cc/archive/master.zip",
+    # sha256 = "55b7c4678b4014be103f0e93eb271858a43493ac7a193ec059289fbdc20b9023",
+)
+
+##################################################
+########        C/C++ DEPENDENCIES        ########
+##################################################
+load("@rules_foreign_cc//:workspace_definitions.bzl", "rules_foreign_cc_dependencies")
```

Comment:
This PR should only contain changes related to the libp2p_helper. Can you remove these dependencies and put them in the appropriate follow-up PR instead?

Reply:
Good catch; thought I had cleaned that out but it was only the libp2p_helper/WORKSPACE.bazel.  Done now.